app/src/App.tsx

```diff
@@ -60,42 +90,105 @@ export const App = () => {
     },
   });
 
-  const onClickCreateButton = () => {
-    result.mutate();
+  const updateTaskMutation = useUpdateTask(
+    {
+      mutation: 
+      {
+        onSuccess: (result) => client.setQueryData(result.id, result.title)
+      },
+    }
+  )
+
+  const onClickCreateTaskButton = useCallback(() => {
+    createTaskMutation.mutate();
     setTitle('');
+  }, [])
+
+  const handleOpenModal = (data: Task) => {
+    setUpdateTask(data);
+    setIsModalOpen(true);
   }
 
-  // const BoxTaskStyle = {
-  //   display: 'grid',
-  //   gridTemplateColumns: '0.3fr 2.4fr 0.3fr'
-  // }
   return (
     <main style={MainStyle}>
+        <EditorModal props={{
+          isModalOpen: isModalOpen,
+          setIsModalOpen: setIsModalOpen,
+          updateTaskMutation: updateTaskMutation,
+          data: updateTask
+        }}/>
```

Comment:
```suggestion
        <EditorModal
          isModalOpen={isModalOpen}
          setIsModalOpen={setIsModalOpen}
          updateTaskMutation={updateTaskMutation}
          data={updateTask}
        />
```
propsで囲わないようにすれば良いと思います！